c/polyjuice.h

```diff
@@ -1010,6 +1020,70 @@ int handle_transfer(gw_context_t* ctx,
   return 0;
 }
 
+int handle_native_token_transfer(gw_context_t* ctx, uint32_t from_id,
+                                 uint256_t value, gw_reg_addr_t* from_addr) {
+  if (g_creator_account_id == UINT32_MAX) {
+    ckb_debug("[handle_native_token_transfer] g_creator_account_id wasn't set.");
+    return ERROR_NATIVE_TOKEN_TRANSFER;
+  }
+  if (!g_eoa_transfer_flag) {
+    ckb_debug("[handle_native_token_transfer] not a native transfer tx");
+    return ERROR_NATIVE_TOKEN_TRANSFER;
+  }
+  evmc_address zero_address = {0};
+  if (memcmp(zero_address.bytes, g_eoa_transfer_to_address.bytes, 20) == 0) {
+    ckb_debug("[handle_native_token_transfer] not allow to transfer to zero address.");
```

Comment:
Example of transfering to zero_address:
https://rinkeby.etherscan.io/tx/0x992e75ad5dbc2b3092ace49d12c95fefb84c3da7f731dea77b71effab797dbbf

Reply:
zero address means g_eoa_transfer_to_address is not set
better idea?